charts/manila-csi-plugin/values.yaml

```diff
@@ -15,6 +15,9 @@ shareProtocols:
  #     dir: /var/lib/kubelet/plugins/cephfs.csi.ceph.com
  #     sockFile: csi.sock
 
+extraLabels:
+  enabled: false
```

Comment:
Is `enabled` needed? Won't just leaving the default value at `{}` do the job?

Reply:
you're right, I'l change it, thanks!


> @JPRbrs thank you for your PR! One comment in-line, otherwise lgtm. Also please add `[manila-csi-plugin] ` prefix to the release note.
> 
> I think it would be good if we kept Cinder's and Manila's charts more or less in sync. Would you mind pushing a follow up PR to add this `extraLabels` to Cinder helm chart too?

Definitely, no problem at all!